diff -ur smstools/Makefile smstools.new/Makefile
--- smstools/Makefile	2005-06-19 08:00:12.000000000 +0000
+++ smstools.new/Makefile	2005-08-17 06:36:55.188942232 +0000
@@ -3,7 +3,7 @@
 VERSION=$(shell grep package_version src/version.h | cut -f2)
 
 compile: 
-	cd src && $(MAKE) -$(MAKEFLAGS) 
+	cd src && $(MAKE) $(MAKEFLAGS) 
 
 install: compile
 	./install.sh
@@ -12,7 +12,7 @@
 	./uninstall.sh
 
 clean:
-	cd src && $(MAKE) -$(MAKEFLAGS) clean
+	cd src && $(MAKE) $(MAKEFLAGS) clean
 
 package: compile clean
 	./package.sh
