diff -burN smstools.orig/Makefile smstools/Makefile
--- smstools.orig/Makefile	2004-04-09 22:24:06.703111488 +0200
+++ smstools/Makefile	2004-04-09 22:26:18.473079408 +0200
@@ -3,13 +3,13 @@
 VERSION=$(shell grep package_version src/version.h | cut -f2)
 
 compile: 
-	cd src && $(MAKE) -$(MAKEFLAGS) 
+	cd src && $(MAKE) $(MAKEFLAGS) 
 
 install: compile
 	./install.sub
 
 clean:
-	cd src && $(MAKE) -$(MAKEFLAGS) clean
+	cd src && $(MAKE) $(MAKEFLAGS) clean
 
 package: compile clean
 	./package.sub
