--- smstools/src/getsms.c.orig	Wed Jan 29 20:36:12 2003
+++ smstools/src/getsms.c	Mon Feb  3 13:22:30 2003
@@ -287,14 +287,14 @@
 {
   int Length;
   int padding;
-  char tmpsender[100];
+  char tmpsender[1000]; /* just in case... this soft looks insecure */
   Length=octet2bin(Pointer);
   padding=Length%2;
   Pointer+=2;
   if ((octet2bin(Pointer) & 112)==80)
   {  // Sender is alphanumeric
     Pointer+=2;
-    sprintf(tmpsender,"%02x%s",Length*4/7,Pointer);
+    snprintf(tmpsender,Length*2,"%02x%s",Length*4/7,Pointer);
     pdu2ascii(tmpsender,sendr);
   }
   else
