diff -ruN smstools/src/smsd.c smstools.org/src/smsd.c
--- smstools/src/smsd.c	Fri Aug 23 21:09:14 2002
+++ smstools.org/src/smsd.c	Tue Sep  3 09:36:22 2002
@@ -756,6 +756,34 @@
   _exit(0);
 }
 
+void daemonize(void)
+{
+	pid_t pid;
+	
+	if ((pid = fork()) < 0) {
+		perror("fork");
+		exit(EXIT_FAILURE);
+	}
+	if (pid > 0)
+		exit(EXIT_SUCCESS);
+
+	if ((pid = setsid()) < 0) {
+		perror("setsid");
+		exit(EXIT_FAILURE);
+	}
+
+	if ((chdir("/")) < 0) {
+		perror("chdir");
+		exit(EXIT_FAILURE);
+	}
+
+	umask(0);
+	
+	close(STDIN_FILENO);
+	close(STDOUT_FILENO);
+	close(STDERR_FILENO);
+}
+
 /* =======================================================================
    Main
    ======================================================================= */
@@ -771,6 +799,8 @@
     signal(SIGTERM,SIG_IGN);
     
   parsearguments(argc,argv);
+  if (printstatus != 1)
+	  daemonize();
   initcfg();
   readcfg();
   logfilehandle=openlogfile("smsd",logfile,LOG_DAEMON,loglevel);  
